src/main/java/uk/gov/companieshouse/api/accounts/validation/CreditorsWithinOneYearValidator.java

```diff
@@ -0,0 +1,191 @@
+package uk.gov.companieshouse.api.accounts.validation;
+
+import java.util.Optional;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.exception.ServiceException;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.creditorswithinoneyear.CreditorsWithinOneYear;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.creditorswithinoneyear.CurrentPeriod;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.creditorswithinoneyear.PreviousPeriod;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.service.CompanyService;
+import uk.gov.companieshouse.api.accounts.transaction.Transaction;
+
+@Component
+public class CreditorsWithinOneYearValidator extends BaseValidator {
```

Comment:
Should we be cross validating against the balance sheet within this class?

Reply:
We have a separate story for cross validation